benchmarks/shelley3pools/fund-addresses.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+. configuration/parameters
+
+export CARDANO_NODE_SOCKET_PATH=${CARDANO_NODE_SOCKET_PATH:-"./logs/sockets/1"}
+NUM_OF_ADDRESSES=${NUM_OF_ADDRESSES:-3}
+CLICMD=${CLICMD:-"../../bin/cardano-cli"}
+WORKDIR=./tmp
+
+# Initial set-up
+if [ -d ${WORKDIR} ]; then rm -rf ./${WORKDIR}; fi
+mkdir -p ./${WORKDIR}/txs
+mkdir -p ./${WORKDIR}/addresses
+
+# Fund address from Genesis
+## Get the initial UTxO TxIn
+${CLICMD} shelley genesis initial-txin \
+          --verification-key-file configuration/genesis/utxo-keys/utxo3.vkey \
```

Comment:
this could be a parameter: "utxo${TXN}"
so the scripts could run in parallel from different genesis addresses